source/includes/_invoice.md

```diff
@@ -4692,8 +4692,46 @@ If successful, returns a status code of 200 and a list of all invoices for this
 
 ### Search invoices
 
-Search for an invoice by a specified search string. If the search string is a number, it is compared to the `invoiceNumber` attribute. An exact match is required. Otherwise, it is compared to the following attributes: `invoiceId`, `accountId`, or `currency`. The operation returns all invoice records in which the search string matches all or part of any one of these attributes.
+Search for an invoice by a specified search string. Search operation can be of two types as follows:
 
+**Basic:**
+
+If the search string is a number, it is compared to the `invoiceNumber` attribute. An exact match is required. Otherwise, it is compared to the following attributes: `invoiceId`, `accountId`, or `currency`. The operation returns all invoice records in which the search string matches all or part of any one of these attributes.
+
+**Advanced**:
+
+Advanced search allows filtering on the specified fields. The prefix marker `_q=1` needs to be specified at the beginning of the search key to indicate this is an advanced query.
+
+Some advanced search key examples:
+* _q=1&status=DRAFT - Returns invoices in `DRAFT` status.
+* _q=1&balance[eq]=200 - Returns invoices with balance=200.
+* _q=1&currency[eq]=USD&balance[gt]100 - Returns invoices where currency is USD and balance is greater than 100.
```

Comment:
Is this supported now?

My original patch only supported search by balance or search by filters:

* https://github.com/killbill/killbill/pull/1995/files/b4d5be55b074cfcdfbfd458a64b95125ed88ab40#r1545224747
* https://github.com/killbill/killbill/blob/fe8dd52e1e1eaafa81dc0d742f89918deba72f06/invoice/src/main/java/org/killbill/billing/invoice/dao/DefaultInvoiceDao.java#L566

Reply:
> Is this supported now?

No sorry, I missed this. I have now amended the docs.